editor/components/rich-text/README.md

```diff
@@ -37,6 +37,22 @@ a traditional `input` field, usually when the user exits the field.
 
 *Optional.* By default, a line break will be inserted on <kbd>Enter</kbd>. If the editable field can contain multiple paragraphs, this property can be set to `p` to create new paragraphs on <kbd>Enter</kbd>.
 
+### `onSetup( editor: Object ): Function`
+
+*Optional.* Called when the editor has been setup.
+
+If you want to use this to give focus to a new editor instance, you must wait until the editor has been rendered before calling `.focus` on it.
```

Comment:
I don't know how common it will be... but if you're dealing with lists of items that have RichText components in them and you want to focus new ones, it is a gotcha. Perhaps that the focus event example in there, because that's an example more people are likely to use, and the note about focusing the editor afterwards?

Reply:
I don't think personally either of these two examples is common because in the last versions of Gutenberg we are trying to do this automatically (handle focus). I know it's not solved entirely for all use-cases. My concern is that people would assume the `onSetup` is only useful to handle focus (in both examples)